CHANGELOG.md

```diff
@@ -8,6 +8,11 @@
     Existing callers should migrate to `Future.doWhile()`.
   * BREAKING CHANGE: Remove IO library. This was deprecated in 2.1.4.
   * BREAKING CHANGE: Remove mirrors library. This was deprecated in 2.1.4.
+  * BREAKING CHANGE: BiMap no longer throws ArgumentError on insertion
+    of a null key or value, if the corresponding K or V type is
+    nullable. As before, values in the map must be unique and
+    ArgumentError is thrown on attempts to add a key-value pair whose
+    value is already in the map.
```

Comment:
Do we need to update the dartdocs about null?

Reply:
I deleted the bits about null from the docs.